src/Cassandra.Tests/Mapping/MappingConfigurationTests.cs

```diff
@@ -22,5 +20,22 @@ public void ConvertTypesUsing_Creates_Uses_MapperFactory_Instance()
             //New instance of the mapper factory
             Assert.AreNotSame(originalMapperFactory, config.MapperFactory);
         }
+
+        [Test]
+        public void Get_FromGlobalConfig_Returns_Mapping_IfExists()
+        {
+            var userMapping = new FluentUserMapping();
+            MappingConfiguration.Global.Define(userMapping);
```

Comment:
Can you use a local instance of `MappingConfiguration`? We should avoid using the global instance on the tests to avoid messing up the other test fixtures.